src/components/Lend/LoanSearch/ChallengeCallout.vue

```diff
@@ -45,7 +45,7 @@
 			</kv-grid>
 		</kv-page-container>
 	</div>
-	<div v-else-if="shareLenderName">
+	<div v-else>
```

Comment:
It looks like this check can be moved to line 56. The `img` tag uses the name for the `alt`.

```
v-if="shareLenderImage && shareLenderName"
```